src/Modules/ModulesGraded.jl

```
@@ -1,5 +1,696 @@
 
 
+###############################################################################
+# Graded Modules constructors
+###############################################################################
+
+@doc Markdown.doc"""
```

```suggestion
@doc raw"""
```
everywhere

---

Thanks for spotting, as far as I see it was the only spot. Should also throw an error by now.